pkgs/development/tools/kdoctor/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, installShellFiles }:
```

One per line, to reduce future diffs:

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, installShellFiles
}:
```